src/service/filtering/filters.py

```diff
@@ -92,6 +95,68 @@ def _gt(num: int, min_: int, name: str):
     return num
 
 
+class BooleanFilter(AbstractFilter):
```

Comment:
This filter needs to be added to FilterSet._FILTER_MAP

Reply:
I know. Coming in future PRs. I think I need to merge the `ColumnType` from `heatmap_common_models` to `columnar_attribs_common_models`. So that we only have one `ColumnType` in the system. But `microtrait` runner is using `heatmap_common_models.ColumnType`. I need to do more testing to make sure `microtrait` can import `columnar_attribs_common_models`

Reply:
Hmm... I don't like the idea of mixing the columns types from heatmaps and attribs. A translator from heatmap -> attribs might be better, only since the filters understand attribs.

Or - have the filters understand both heat map and attribs column types

Reply:
👍

Reply:
:+1: